RULES.md

```diff
@@ -38,13 +40,13 @@ The goal of the Training Algorithm Track is to reach the same results faster ("t
 
 The intention is that a training algorithm submission will be broadly applicable and useful without customization to the specific [workload](#workloads) (model, dataset, loss function). We want to discourage detecting the particular workload and doing something highly specific that isn't generally useful. In order to further discourage submissions that overfit to the particular [public benchmark workloads](#public-workloads), submissions must also perform well on one or more [held-out workloads](#held-out-workloads) released after the submission deadline.
 
 ### Submissions
 
-A valid submission is a piece of code with the same high-level structure as a reference implementation that can train all benchmark workloads on the [competition hardware](#competition-hardware) (defined in the [Scoring](#scoring) Section but ultimately in the call for submissions). The validation set performance will be checked regularly during training (see the [Evaluation during training](#evaluation-during-training) Section) and training halts when a workload-specific target error has been reached. For each workload, the training time to reach this (validation set) target error will be used as an input to the [scoring process](#scoring) for the submission. Additionally, the test set performance will be probed using the final model parameters to confirm that it also reaches a slightly more generous target performance on this unseen data. Submissions using [external tuning](#external-tuning-ruleset) will be tuned independently for each workload using a single workload-agnostic search space for their specified hyperparameters. Submissions under either tuning ruleset may always self-tune while on the clock.
+A valid submission is a piece of code with the same high-level structure as a reference implementation that can train all benchmark workloads on the [competition hardware](#competition-hardware) (defined in the [Scoring](#scoring) section but ultimately in the call for submissions). The validation set performance will be checked regularly during training (see the [Evaluation during training](#evaluation-during-training) section) and training halts when a workload-specific [target error](#defining-target-performance) has been reached. For each workload, the training time to reach this (validation set) target error will be used as an input to the [scoring process](#scoring) for the submission. Additionally, the test set performance will be probed using the final model parameters to confirm that it also reaches a slightly more generous target performance on this unseen data (see the [scoring](#scoring)] section for details). Submissions using [external tuning](#external-tuning-ruleset) will be tuned independently for each workload using a single workload-agnostic search space for their specified hyperparameters. Submissions under either tuning ruleset may always self-tune while on the clock.
```

Comment:
> A valid submission

is a piece of code **that defines all of the submission functions**

> (defined in the [Scoring](#scoring) section but ultimately in the call for submissions)

Can we just leave it as either "defined in the scoring section" or "to be specified in the call for submissions"? I guess the latter is preferable if we expect the competition hardware to change from call to call.

Nit: it might be more consistent to drop "competition" and use "benchmark" throughout the rules. (Is the competition the same as the benchmark?)

Reply:
@mikerabbat Do I understand it correctly that you would like to change the sentence:

"A valid submission is a piece of code with the same high-level structure as a reference implementation that can train all benchmark workloads on the [competition hardware](#competition-hardware) (defined in the [Scoring](#scoring) section but ultimately in the call for submissions)."

to something like:

"A valid submission is a piece of code that defines all of the submission functions and is able to train all benchmark workloads on the [competition hardware](#competition-hardware) (defined in the [Scoring](#scoring) section)."?

I would expect us to update this rules document for each iteration of the benchmark and so we could update the competition hardware (or should I say: benchmark hardware) from call to call in this doc.

Reply:
I consistently use "benchmark" now instead of "competition", including calling the the "benchmarking hardware" and the 
(scalar) "benchmarking score". Thanks for the suggestion!

Reply:
> to something like:
> 
> "A valid submission is a piece of code that defines all of the submission functions and is able to train all benchmark workloads on the [competition hardware](https://github.com/mlcommons/algorithmic-efficiency/pull/117#competition-hardware) (defined in the [Scoring](https://github.com/mlcommons/algorithmic-efficiency/pull/117#scoring) section)."?

Yes, exactly!